src/main/java/org/alliancegenome/curation_api/model/ingest/dto/fms/BioSampleAgeDTO.java

```diff
@@ -0,0 +1,13 @@
+package org.alliancegenome.curation_api.model.ingest.dto.fms;
+
+import org.alliancegenome.curation_api.model.ingest.dto.base.BaseDTO;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class BioSampleAgeDTO extends BaseDTO {
+	private WhenExpressedDTO stage;
```

Comment:
I know this isn't your code, but could we rename the WhenExpressedDTO and WhereExpressedDTO classes to WhenExpressedFmsDTO and WhereExpressedFmsDTO respectively.  That makes it clear that these are not expected to align with the LinkML schema.